docs/static/includes/sample.telem.tf

```
@@ -1,5 +1,60 @@
 locals {
-  module_version = "v1.0.0"
+  # This is the unique id for the module that is supplied by the AVM team.
+  # TODO: change this to the PUID for the module. See https://azure.github.io/Azure-Verified-Modules/specs/shared/#id-sfr3---category-telemetry---deploymentusage-telemetry
+  telem_puid = "00000000"
```

```suggestion
  telem_puid = "46d3xtf"
```

is this not the value I suggest as per: https://azure.github.io/Azure-Verified-Modules/specs/shared/#id-sfr3---category-telemetry---deploymentusage-telemetry:~:text=Terraform%20%3D%3D%2046d3xtf.res.compute%2Dvirtualmachine.v1%2D2%2D3.eum3